Project.toml

```diff
@@ -1,17 +1,20 @@
 name = "ChainRulesCore"
 uuid = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
-version = "0.5.2"
+version = "0.5.3"
 
 [deps]
 MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
```

Comment:
Can be removed here as already in extras :)

Reply:
Thanks!